examples/demo13.py

```diff
@@ -54,308 +55,302 @@
-
-            # Have to find where to place it:
-            for i_gal_use in range(i_gal*n_tot//n_use, (i_gal+1)*n_tot//n_use):
-                # Account for the fractional part of the position:
-                ix = int(math.floor(x_stamp[i_gal_use]+0.5))
-                iy = int(math.floor(y_stamp[i_gal_use]+0.5))
-                # We don't actually use this offset.
-                offset = galsim.PositionD(x_stamp[i_gal]-ix, y_stamp[i_gal]-iy)
-
-                # Create a nominal bound for the postage stamp given the integer part of the
-                # position.
-                stamp_bounds = galsim.BoundsI(ix-0.5*stamp_size, ix+0.5*stamp_size-1,
-                                              iy-0.5*stamp_size, iy+0.5*stamp_size-1)
-
-                # Find the overlapping bounds between the large image and the individual postage
-                # stamp.
-                bounds = stamp_bounds & final_image.bounds
-
-                # Just to inject a bit of variety into the image, so it isn't *quite* as obvious
-                # that we've repeated the same 10 objects over and over, we randomly rotate the
-                # postage stamp by some factor of 90 degrees and possibly include a random flip.
-                if flip_stamp[i_gal_use] > 0.5:
-                    new_arr = numpy.ascontiguousarray(
-                        numpy.rot90(stamp.array, n_rot_stamp[i_gal_use]))
-                else:
-                    new_arr = numpy.ascontiguousarray(
-                        numpy.fliplr(numpy.rot90(stamp.array, n_rot_stamp[i_gal_use])))
-                stamp_rot = galsim.Image(new_arr, scale=stamp.scale)
-                stamp_rot.setOrigin(galsim.PositionI(stamp_bounds.xmin, stamp_bounds.ymin))
-
-                # Rescale the flux to match that of a randomly chosen galaxy in the galaxy, but
-                # keeping the same SED as for this particular galaxy.  This gives a bit more
-                # variety in the flux values and SNR of the galaxies in the image without having
-                # to render images of many more objects.
-                flux_scaling = 10**(-0.4*(mag_stamp[i_gal_use]-mag_list[i_gal]))
-
-                # Copy the image into the right place in the big image.
-                final_image[bounds] += flux_scaling * stamp_rot[bounds]
-
-        # Now we're done with the per-galaxy drawing for this image.  The rest will be done for the
-        # entire image at once.
-        logger.info('Postage stamps of all galaxies drawn on a single big image for this filter.')
-        logger.info('Adding the sky level, noise and detector non-idealities.')
+        # GalSim uses the term Bandpass for the class that defines the throughput across a
+        # filter bandpass, partly because "filter" is a reserved word in python.  So we follow
+        # that convention here as well.
+        bandpass = roman_filters[filter_name]
+        y_bandpass = roman_filters['Y106']
```

Comment:
Not a big deal, but why do this inside the loop?

Reply:
It's not really an inefficiency, since it's just a dict access.  But I'll pull it out of the loop and add a comment about how we'll be using it.